### DISCUSSION TITLE
Installation on Linux Mint

### DISCUSSION THREAD

**Discussion by SHU-red** [pear-devs/pear-desktop#91] [Q&A] [answered]
Hello everyone,
this may be a dumb question:

Can i somehow install this software on my Linux Mint System and receive updates without making my own build?

I think i would need a .deb package or something?

Thanks a lot

**Comment by th-ch** [comment, marked as answer]
Hey @SHU-red, thanks for the question! I added new Linux targets, you can find a [.deb](https://github.com/th-ch/youtube-music/releases/download/v1.7.5/youtube-music_1.7.5_amd64.deb) in the last pre-release, would that solve your issue?

**Reply by SHU-red** [reply]
Oh yes man!
Works awsome!

Thanks a lot and i already like how it works!

Keep up the good work!

**Comment by inson1** [comment]
@SHU-red What about closing this discussion, if it is resolved? :)

**Reply by SHU-red** [reply]
Absolutely!
Thanks ...
